doc/nrf/ug_matter_overview_network_topologies.rst

```diff
@@ -1,32 +1,31 @@
 .. _ug_matter_overview_network_topologies:
 
-Matter network overview
-#######################
+Matter network topology and concepts
+####################################
 
 .. contents::
    :local:
    :depth: 2
 
-Matter network can be composed of Ethernet, Wi-Fi, and Thread devices.
-Networks are joined together, which means that devices that belong to different network can talk to each other within the local Matter fabric (infrastructure).
-All communication is achieved over IPv6, although the Matter network is capable of running in the absence of IPv6 infrastructure connected to the Internet.
+A Matter network can be composed of Ethernet, Wi-Fi, and Thread devices, which in turn can be part of one or more such IPv6-based networks.
+Matter joins these devices together within the local Matter fabric (infrastructure), which allows devices that belong to different network to talk to each other.
```

Comment:
That's not entirely true. All devices connected to the different networks (ethernet, Wi-Fi and Thread), but connected to one IP network are able to talk to each other over IP even without Matter. I would suggest to rephrase this sentence a little bit:
```suggestion
Matter joins these devices together within the local Matter fabric (infrastructure), which allows devices to talk to each other using the same Matter application layer, despite using different network technologies underneath.
```